### PR TITLE
Add poll value command

### PR DESCRIPTION
**Pull request by marcelveldt**
This command was added in Zwave JS 6.1.0

Might come in handy for HA so users can eventually poll values manually if that is really required.
I'll expose it with a service call (with a big fat warning)

### CODE REVIEW

File: src/lib/node/message_handler.ts [hunk 1]
```diff
@@ -31,6 +31,8 @@ export class NodeMessageHandler {
       case NodeCommand.abortFirmwareUpdate:
         await node.abortFirmwareUpdate();
         return {};
+      case NodeCommand.pollValue:
+        return await node.pollValue(message.valueId);
```
Comment (balloob) [inline src/lib/node/message_handler.ts:35]: The return value should always be wrapped in an object. `{ value: X }`. 

Also make sure that the result value can be JSON serialized.

Comment (marcelveldt) [reply src/lib/node/message_handler.ts:35]: I was not sure if this actually returns the value as the docs are a bit unclear. I'll inspect the code to find out.

**Comment by marcelveldt** [comment]
I've adjusted it. As the value's type is unknown, I've set it to any.

File: src/lib/node/message_handler.ts [hunk 2]
```diff
@@ -31,6 +31,9 @@ export class NodeMessageHandler {
       case NodeCommand.abortFirmwareUpdate:
         await node.abortFirmwareUpdate();
         return {};
+      case NodeCommand.pollValue:
+        const result = await node.pollValue<any>(message.valueId);
```
Comment (balloob) [inline src/lib/node/message_handler.ts:35]: So it looks like this returns the updated value. So this should be called `value` and not `result` ?

Comment (marcelveldt) [reply src/lib/node/message_handler.ts:35]: It *may* return the value if the node responds and polling is supported in the CC.
But yeah, I could rename it to value

**Review by balloob — APPROVED** [review]

**Pull request merged**